manifests/redis/external/munin_node_plugin.pp

```diff
@@ -1,5 +1,5 @@
 class ducktape::redis::external::munin_node_plugin(
-  Boolean $enabled = true,
+  Boolean $enabled,
   Hash $instance_defaults = {
       bind => $::redis::bind,
       port => $::redis::port,
```

Comment:
How can you represent these variables on the yaml file?

Im guessing
```yaml
ducktape::redis::external::munin_node_plugin::instance_defaults
  bind: lookup(...)
  port: lookup(...)
```

Reply:
Your guess is half right, but again you can't rely on other modules defining it's params on hiera

Reply:
Déjalo en la clase.